src/Application/System/Commands/SeedDataCommand.cs

```diff
@@ -1901,11 +1901,11 @@ private async Task CreateOrUpdateItems(CancellationToken cancellationToken)
                     .ToArrayAsync(cancellationToken);
                 foreach (var userItem in userItems)
                 {
-                    int price = _itemModifierService.ModifyItem(userItem.Item!, userItem.Rank).Price;
```

Comment:
This code is reimbursing users when a item was sold. You don't need to call ModifyItem item which used to upgrade the item to know its final cost.

Not sure if it should be done in this PR though.

Reply:
Correct me if i'm wrong , but the item modifier service should be completely removed.
I planned on doing this in a following pr

Reply:
Yep, it was just a heads up